### ISSUE TITLE
[Feature Request] Improve constrast of checkboxes

### ISSUE THREAD

**Issue by PurpleMyst** [darkreader/darkreader#2610]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to turn off dark reader to be able to tell if a checkbox is checked on github and on similiar sites

**Describe the solution you'd like**
I'd like if checkboxes were somehow given additional constrast in the Dynamic Theme

**Additional context**
Here is a screenshot of how a checkbox currently looks on GitHub:
![Screenie](https://files.catbox.moe/cwslgn.png)

**Comment by Gusted** [comment]
Example URL?

**Comment by PurpleMyst** [comment]
Um I can just make a checkbox:

- [x] <-- this is checked
- [ ] <-- this is not checked

it is very hard to tell which is which to my eyes

**Comment by PurpleMyst** [comment]
Also here's the link to where I took the screenshot: https://github.com/rust-analyzer/rust-analyzer/issues/223#issuecomment-578559627

**Comment by Gusted** [comment]
![image](https://user-images.githubusercontent.com/25481501/82842527-30fa7180-9eda-11ea-8de6-5cd2a78f9b7b.png)
Which filterSettings do you have?
Which OS do you use?

**Comment by Gusted** [comment]
![](https://i.imgur.com/dmEvWwH.png)

Potential fix

**Comment by Gusted** [comment]
BTW we can't target all checkboxes as on other sites some have custom CSS for it and INVERT it would interfere with it. So if you only notice it on Github it's fine.

**Comment by PurpleMyst** [comment]
I seem to have only noticed in on Github but that might be because it's the only place that I regularly see disabled checkboxes

**Comment by PurpleMyst** [comment]
> 
> 
> ![image](https://user-images.githubusercontent.com/25481501/82842527-30fa7180-9eda-11ea-8de6-5cd2a78f9b7b.png)
> Which filterSettings do you have?
> Which OS do you use?

I'm on Windows with these Dynamic filterSettings:
![](https://i.imgur.com/upiAuiR.png)

**Comment by PurpleMyst** [comment]
Any movement on this? The solution @Gusted posted looked pretty nice.

**Comment by sprinkledcheese** [comment]
![chrome_7MT7bG5x0m](https://user-images.githubusercontent.com/65959962/83262559-d2cfd600-a1da-11ea-94df-4c2199c8e39b.png)
This only happens with enabled/disabled attributes.

**Comment by PurpleMyst** [comment]
Yep, you might not've seen it but I made an edit to the issue where I said that the issue is with the disabled ones

**Comment by Gusted** [comment]
It's a more global issue and seem's like our inject CSS isn't styling checkboxes :eyes:

**Comment by cryptosteve2** [comment]
For me in the AdGuard Home settings it looks like this:

![20221004@213710](https://user-images.githubusercontent.com/6260784/193910130-4e5ec6d3-86ff-43c6-bc7f-bb85e423c570.png)

**Comment by Gusted** [comment]
@cryptosteve2 That's another issue, not related to this one.

**Comment by wan7teen-ping** [comment]
Hi I will help you with everything you need but you have to subscribe to
the youtube channel and I will fulfill all your requests

https://youtu.be/aIaqr4eq18g

Pada Rab, 5 Okt 2022, 3:39 AM cryptosteve2 ***@***.***>
menulis:
